dashboard/build/prometheus/dashboard.libsonnet

```diff
@@ -18,8 +18,7 @@ dashboard_raw(
   name='PROMETHEUS_JOB',
   label='Job',
   type='constant',
-  pluginId=null,
-  pluginName=null,
+  value='tarantool',
```

Comment:
As far as I understand this is the default value for a user input? Does it make sense to change it too with the JOB value?

Reply:
Templating dashboards are built as part of tests, they do not have separate `make` tests and are built with libsonnet module imports.